app/models/video.rb

```diff
@@ -0,0 +1,7 @@
+class Video < ActiveRecord::Base
+  attr_accessible :description, :title, :youtube_id
+
+  def youtube_link
+    'https://www.youtube.com/watch?v=' + self.youtube_id
```

Comment:
Embedded Ruby would be prettier here. Use `"https://www.youtube.com/watch?v=#{self.youtube_id}"`.